cmd/api/main.go

```diff
@@ -87,10 +88,9 @@ func main() {
 
 	dbURI := fmt.Sprintf("host=%s user=%s password=%s dbname=%s port=%s sslmode=%s", configFile.DB_HOST, configFile.DB_USER, configFile.DB_PASSWORD, configFile.DB_NAME, configFile.DB_PORT, configFile.DB_SSLMODE)
 
-	db, err := gorm.Open(postgres.Open(dbURI), &gorm.Config{})
-	if err != nil {
-		log.Fatalf("Failed to open the results.db: %v", err)
-	}
+	db, err := gorm.Open(postgres.Open(dbURI), &gorm.Config{
+		Logger: logger.Default.LogMode(logger.Silent),
+	})
```

Comment:
Does this mean that the GORM logs are permanently suppressed? If yes, can we use the zap log level to turn these back on? For example - log if "debug" is set.

Also Need to ensure that we catch the error opening the database connection and fail:

```suggestion
	db, err := gorm.Open(postgres.Open(dbURI), &gorm.Config{
		Logger: logger.Default.LogMode(logger.Silent),
	})
	if err != nil {
		log.Fatalf("Failed to open the results.db: %v", err)
	}
```

Reply:
@AndrienkoAleksandr following up on our slack conversation - I think for now we can suppress these log messages. I will create a follow up issue to make these logs configurable.

Reply:
Hello, default log library doesn't have log level feature, so if you don't mind I updated my pr to use zap logger and I made log level configurable using config file...

Reply:
That's fine, though I suspect we will need to propagate this logger to other spots in the code.